spec/sidekiq/schema_contract/delete_validation_records_job_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative Rails.root.join('app', 'models', 'schema_contract', 'validation_initiator')
+
+RSpec.describe SchemaContract::DeleteValidationRecordsJob do
+  let(:job) { described_class.new }
+  let(:response) do
+    OpenStruct.new({ success?: true, status: 200, body: { key: 'value' } })
+  end
+
+  context 'when records exist that are over a month old' do
+    before do
```

Comment:
I would generally recommend using `let` over ivars in rspec. For a case like this, you could use `let!` to achieve the same effect as a before block.

Reply:
Ah, didn't know `let!` would do what I wanted. I did `let` and it didn't work, so TIL. Changed!